src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -1246,29 +1256,14 @@ def _get_acr_from_image(cmd, app):
         app.registry_server = app.image.split("/")[
             0
         ]  # TODO what if this conflicts with registry_server param?
+
         parsed = urlparse(app.image)
         registry_name = (parsed.netloc if parsed.scheme else parsed.path).split(".")[0]
-        if app.registry_user is None or app.registry_pass is None:
```

Comment:
what if only app.registry_user provided?

Reply:
It will be handled in `create_containerapp` or `update_containerapp`, we will look up creds for it, also I has test case cover it: test_containerapp_registry_acr_look_up_credentical